packages/react/src/components/Forms/Select/test-utils.tsx

```diff
@@ -49,3 +49,18 @@ export const expectSelectedOptionsText = (expectedOptions: string[]) => {
   const valueElement = document.querySelector(".c__select__inner__value");
   expect(valueElement?.textContent).toEqual(expectedOptions.join(", "));
 };
+
+export const expectLoaderToBeVisible = async () => {
+  await waitFor(() => {
+    const loader = screen.queryByRole("status", {
+      name: "Loading data",
+    });
+    expect(loader).toBeVisible();
+  });
+};
+export const expectLoaderNotToBeInTheDocument = async () => {
+  await waitFor(() => {
+    const loader = screen.queryByRole("status");
+    expect(loader).not.toBeInTheDocument();
+  });
+};
```

Comment:
In the first method you are using name attribute and in the second nope.

You should homogeneize this two helpers. IMO, if you want to use name attribute to prevent to catch several loaders in some scenarii, you must allow to pass a name as argument.